switchapi/es2k/switch_pd_lag.c

```diff
@@ -154,11 +154,11 @@ switch_status_t switch_pd_tx_lag_table_entry(switch_device_t device,
   }
 
   status = tdi_data_field_id_with_action_get(
-      table_info_hdl, LNW_ACTION_SET_EGRESS_PORT_PARAM_EGRESS_PORT, action_id,
+      table_info_hdl, ACTION_SET_EGRESS_PORT_PARAM_EGRESS_PORT, action_id,
       &data_field_id);
   if (status != TDI_SUCCESS) {
     krnlmon_log_error("Unable to get data field id param for: %s, error: %d",
-                      LNW_ACTION_SET_EGRESS_PORT_PARAM_EGRESS_PORT, status);
+                      LNW_ACTION_FWD_TO_VSI_PARAM_PORT, status);
```

Comment:
Shouldn't this identifier be the same as the one used in the preceding function call?